packages/teams-js/src/public/navigation.ts

```diff
@@ -33,14 +33,14 @@ export function returnFocus(navigateForward?: boolean): void {
  */
 export function navigateToTab(tabInstance: TabInstance, onComplete?: onCompleteHandlerFunctionType): void {
   ensureInitialized(runtime);
-  onComplete = onComplete ? onComplete : getGenericOnCompleteHandler();
+  const completionHandler: onCompleteHandlerFunctionType = onComplete ?? getGenericOnCompleteHandler();
```

Comment:
>const completionHandler [](http://example.com/codeflow?start=2&length=23)

The changes in this file are needed for the compiler to recognize that the completion handler function will never be undefined. It was also just generally unusual practice to redefine the value of input parameters, in the previous approach.